mlflow/sklearn/__init__.py

```diff
@@ -819,6 +823,14 @@ def f(self, *args, **kwargs):
     estimators_to_patch = set(estimators_to_patch).union(
         set(_get_meta_estimators_for_autologging())
     )
+
+    # from sklearn.utils.testing import all_estimators
```

Comment:
Can we remove this commented code?

Reply:
I thought I have deleted it lol. Will be more careful next time!